From 07ef071f5fe0d623a96865d50f3502943396f19c Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 24 Jan 2014 12:54:47 +0100
Subject: [PATCH 058/509] spi: zynq: : Update license header

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit 2f8d0c78f512d3d01331273715a7aaf40d49fb44

Add 2014 to header and fix it to unified name.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/spi/spi-zynq.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/spi/spi-zynq.c b/drivers/spi/spi-zynq.c
index c810d75..5dd3d09 100644
--- a/drivers/spi/spi-zynq.c
+++ b/drivers/spi/spi-zynq.c
@@ -1,8 +1,7 @@
 /*
- *
  * Xilinx Zynq SPI controller driver (master mode only)
  *
- * (c) 2008-2011 Xilinx, Inc.
+ * Copyright (C) 2008 - 2014 Xilinx, Inc.
  *
  * based on Blackfin On-Chip SPI Driver (spi_bfin5xx.c)
  *
@@ -16,7 +15,6 @@
  * Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-
 #include <linux/clk.h>
 #include <linux/delay.h>
 #include <linux/interrupt.h>
-- 
1.7.5.4

